src/Resources/Guest/README.md

```diff
@@ -0,0 +1,14 @@
+# Guest Resources
+
+## Available sub resources:
+- [Basket](###Basket)
+
+## Example usage:
+
+## **Basket**
+| Function call                       | Action                                      |
+|-------------------------------------|---------------------------------------------|
+| `client.guest.basket.get()`         | Get a guest's basket                        |
+| `client.guest.basket.addLine()`     | Add a basketLine to a guest's basket        |
+| `client.guest.basket.updateLine()`  | Update a basketLine in a guest's basket     |
+| `client.guest.basket.removeLine()`  | Remove a basketLine from a guest's basket   |
```

Comment:
I like this, but I'm thinking it will quickly fall out of sync with the code.

Perhaps in the short future, it may be an idea to look at auto-documentation tools, generated from the code itself.